pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetShouldDisplayLanguageChangePopupRoute.java

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.ddp.route;
+
+import org.apache.http.HttpStatus;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.StudyDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.json.errors.ApiError;
+import org.broadinstitute.ddp.model.study.StudySettings;
+import org.broadinstitute.ddp.util.ResponseUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class GetShouldDisplayLanguageChangePopupRoute implements Route {
```

Comment:
After updating the `GetStudyDetailRoute`, can you also update the docs in `pepper-apis/docs/specification`? It's in YAML / Swagger / OpenAPI format, but ping me if you need help with that.